### PR TITLE
Update index-state and nix updates

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: nix/default.nix [hunk 1]
```diff
@@ -2,28 +2,30 @@
 , crossSystem ? null
 , config ? {}
 , sourcesOverride ? {}
+, gitrev ? null
```
Comment (coot) [inline nix/default.nix:5]: I don't see it used anywhere, is it really needed?

Comment (newhoggy) [reply nix/default.nix]: Not used.  Removed.  Thanks!

**Comment by mrBliss** [comment]
@coot Any reason not to merge this (or are we waiting until after the hard fork)?

**Comment by coot** [comment]
@nc6 are you sure we can already merge PRs?

**Pull request merged**